src/web/app/components/question-types/question-statistics/question-statistics-calculation/constsum-recipients-question-statistics-calculation.ts

```diff
@@ -54,7 +63,10 @@ export class ConstsumRecipientsQuestionStatisticsCalculation
       const sum: number = answers.reduce((a: number, b: number) => a + b, 0);
       this.totalPointsPerOption[recipient] = sum;
       this.averagePointsPerOption[recipient] = +(answers.length === 0 ? 0 : sum / answers.length).toFixed(2);
+      this.averagePointsExcludingSelf[recipient] =
+          +(answers.length === 0 ? 0 : (sum - pointsPerOptionToSelf[recipient]) / answers.length).toFixed(2);
```

Comment:
Slight issue with the following, didn't realise it myself until testing it either. Looking at Gene's line, the average excluding self ought to be 50, but it's 25, because it's incorrectly dividing by 2 instead of 1:

<img width="1237" alt="Screenshot 2021-06-30 at 1 27 50 AM" src="https://user-images.githubusercontent.com/41856541/123841658-64f85f80-d942-11eb-96b8-6e08513eb4fb.png">

Will need to find a way to determine if `pointsPerOptionToSelf[recipient]` has actually been assigned a value or not. One way (if it's possible) is to allow it to be undefined instead of setting it to 0 by default above, then here use a conditional to check if it's been assigned a value -- then subtract and divide accordingly

Alternatively, could also directly capture the excluding-self-scores using `pointsPerOptionExcludingSelf`, `answersExcludingSelf`, `sumExcludingSelf` etc. instead of having to minus it off at the end

Reply:
This bug should hopefully be fixed now!